.pre-commit-config.yaml

```diff
@@ -26,13 +26,16 @@ repos:
             'flake8-no-pep420==2.0.0',
         ]
         args: ["--config=.flake8"]
```

Comment:
changing this one to `  args: ["--config=backend/.flake8"]` should fix file exclusion problem, and consequently there will be no need to duplicate the regex.